### ISSUE TITLE
slipping

### ISSUE THREAD

**Issue by ozgali** [mixxxdj/mixxx#13197]
### Bug Description

not sure if just me but I find in version 2.4 when tracks are synced they will start slipping out of sync. Never had this in pervious versions.

### Version

2.4

### OS

windows

**Comment by daschuer** [comment]
Which file types (file extensions) are effected?
Which exact windows version (update) do you use?

**Comment by ozgali** [comment]
Hi,I'm using windows 10 vers 22H2. They are MP3 and WAV files but unsure if both or just one is drifting. I'm also using a newmark platinum FX controller and find the mapping for the brows/selector knob goes a little haywire. The knob will start jumping 4 songs at a time in the song list after using it for a little while.
GALI 

    On Friday, 3 May 2024 at 12:15:00 am AWST, Daniel Schürmann ***@***.***> wrote:  
 
 


Which file types (file extensions) are effected?
Which exact windows version (update) do you use?

**Comment by ozgali** [comment]
I try not to use sync and just adjust the bpm via slide
GALI 

    On Monday, 6 May 2024 at 12:56:05 pm AWST, Daniel Schürmann ***@***.***> wrote:  
 
 


You use sync, with a latches sync button (long press) right?

Are you able to record the issue? Is it visible also in the waveform beats?

**Comment by ronso0** [comment]
> I try not to use sync and just adjust the bpm via slide

So you're suing the BPM display of the decks which show BPM _rounded to two decimals_ which can of course cause drift.
Exactly like on real turntables, you have to re-adjust the pitch if tracks are supposed to run in parallel for longer than a quick transition.

Closing this as 'Won't fix' since that's the way it is.
If you use Sync (latched) and Quantize on two decks and the tracks still drift, feel free to reopen this issue.
Thank you!
